.changeset/breezy-cows-change.md

```diff
@@ -0,0 +1,6 @@
+---
+"@wso2is/react-components": patch
+"@wso2is/theme": patch
```

Comment:
Please include all the dependent packages as well in the changeset. You can view the dependents for each module by running `npx nx graph` in the project root

Reply:
This is to cover the changes done with https://github.com/wso2/identity-apps/pull/4657.